includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -448,6 +449,7 @@ public function process_payment( $order_id ) {
 		return [
 			'result'         => 'success',
 			'payment_needed' => $payment_needed,
+			'order_id'       => $order_id,
```

Comment:
We are returning the order ID inside the `redirect_url` parameter returned here, so I considered pulling this from the URL on the client instead of including it twice, so we could send it back if there was an error, but in the end this just seemed simpler than adding additional code to parse the URL in Javascript.